src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
One thing I mentioned in [1299](https://github.com/cBioPortal/cbioportal/issues/1299) that we need to support at MSK currently and perhaps other institutes in the future is running multiple instances of the cBioPortal concurrently.  This could get hairy if we rely on environment variables.